### DISCUSSION TITLE
NVIDIA Async Reprojection?

### DISCUSSION THREAD

**Discussion by ghost** [alvr-org/ALVR#1213] [Q&A] [answered]
Since there is now support for it in the drivers, would it be possible for it to work on NVIDIA cards? I feel like it helps reduce the amount of jutter by a massive amount.

**Comment by zmerp** [comment, marked as answer]
Have you tried disabling the code that force disables async reprojection?

**Reply by zmerp** [reply]
I see now you opened the other issue. The problem is that it's SteamVR that needs to add support to async reprojection, not ALVR.
